src/database.js

```diff
@@ -6,56 +6,66 @@ const { DB_USER, DB_PASSWORD, DB_HOST, DB_PORT, DB_NAME } = process.env
 const sequelize = new Sequelize(`postgres://${DB_USER}:${DB_PASSWORD}@${DB_HOST}:${DB_PORT}/${DB_NAME}`, {
   logging: false,
   native: false
```

Comment:
Why do we need it?

Reply:
logging: A function that gets executed every time Sequelize would log something. Function may receive multiple parameters but only first one is printed by console.log. To print all values use (...msg) => console.log(msg). Set to console.log to see the raw SQL queries.
-
native: A flag that defines if native library shall be used or not. Currently only has an effect for postgres. Lets Sequelize know we can use pg-native for ~ 30% more speed. In the case of 'pg' -- set this to true will allow SSL support.